src/lifecycle/properties-init.js

```diff
@@ -97,20 +104,11 @@ function createNativePropertyDefinition (name, opts) {
 
     info.updatingProperty = true;
 
-    if (info.hasBeenSetOnce) {
```

Comment:
The `init` function takes care of the initial setting, so no need to set this anymore.